tasks/ubuntu/bionic.task/os_boot.erb

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Does this file differ from `ubuntu.task/os_boot.erb`? This file should also be inherited, rather than overwritten, if possible.